CHANGELOG

```diff
@@ -1,5 +1,6 @@
-Pending:
+2022-31-01 - 1.2.0
 
+- Ruby 2.6 is EOL support removed
```

Comment:
It looks like we should remove Ruby 2.6 from our Ruby test matrix as part of this change: https://github.com/discourse/mini_sql/blob/bd46c55f1b9600f58e015c7720d19a6a8ffcbea3/.github/workflows/ci.yml#L40